ruff.toml

```diff
@@ -22,7 +22,5 @@ ignore = [
 ]
 
 [format]
-# Enable preview, required for quote-style = "preserve"
-preview = true
 # https://docs.astral.sh/ruff/settings/#format-quote-style
```

Comment:
```suggestion
# https://docs.astral.sh/ruff/settings/#format_quote-style
```
Let's maybe fix the link in this PR, since it is logically connected to it.